core-portal/css/cms.css

```
@@ -0,0 +1,3 @@
+.portal-logo {
+  padding-block: 10px;
```

Use height, like other portals.

```suggestion
  height: 30px;
```

Easier to understand.